proposed/extended-coding-style-guide-meta.md

```diff
@@ -46,22 +46,18 @@ This PSR shares the same goals as PSR-2.
 > of guidelines to be used among all those projects. Thus, the benefit of this guide is
 > not in the rules themselves, but in the sharing of those rules.
 
-This PSR is an extension of PSR-2, and therefore also an extension of PSR-1. It shall be
-included in the document that compliance with PSR-N implicitly requires compliance with
-PSR-2 and PSR-1.
+This PSR is an extension of PSR-2, and therefore also an extension of PSR-1. The basis of
+PSR-12 is PSR-2 and therefore a list of differences is provided below to assist with migration
+but it should should be considered as a different specification.
```

Comment:
Lots of "therefore"s in here where they probably shouldn't be:
> This PSR is an extension of PSR-2 and therefore an extension of PSR-1. Because PSR-12 replaces PSR-2, a list of differences is provided below to assist with migration.